llama_stack/providers/remote/inference/vllm/vllm.py

```diff
@@ -136,6 +141,51 @@ def _convert_to_vllm_finish_reason(finish_reason: str) -> StopReason:
     }.get(finish_reason, StopReason.end_of_turn)
 
 
+async def _process_vllm_chat_completion_stream_response(
```

Comment:
Need to merge with the existing `process_chat_completion_stream_response` to pass other tests